src/neo/SmartContract/InteropService.Blockchain.cs

```diff
@@ -45,8 +45,25 @@ private static bool Blockchain_GetBlock(ApplicationEngine engine)
 
             private static bool Blockchain_GetTransaction(ApplicationEngine engine)
             {
-                ReadOnlySpan<byte> hash = engine.CurrentContext.EvaluationStack.Pop().GetSpan();
-                Transaction tx = engine.Snapshot.GetTransaction(new UInt256(hash));
+                Transaction tx;
+                StackItem item = engine.CurrentContext.EvaluationStack.Pop();
+
+                if (item.IsNull)
+                {
+                    if (engine.ScriptContainer is Transaction senderTx)
```

Comment:
I think it's not very convenient for developers to get the current transaction by passing a `null` first. How about using `System.Blockchain.GetCurrentTransaction`?

Reply:
It's fine for me